flink-core/src/main/java/org/apache/flink/api/common/io/ReplicatingInputFormat.java

```diff
@@ -133,4 +133,18 @@ public RuntimeContext getRuntimeContext(){
 			throw new RuntimeException("The underlying input format to this ReplicatingInputFormat isn't context aware");
 		}
 	}
+	
+	@Override
+	public void openInputFormat() {
+		if(this.replicatedIF instanceof RichInputFormat){
```

Comment:
Can you add a space before and after the parentheses of the condition, i.e., `if (condition) {`?
Please check the other changes as well.